pynear/tests/integration/test_cli.py

```diff
@@ -0,0 +1,197 @@
+import json
+import os
+import random
+
+import delegator
+import pytest
+from retrying import retry
+
+from near.pynear.lib import NearLib
+
+nearlib = NearLib('http://localhost:3030/')
+
+
+@retry(stop_max_attempt_number=5, wait_fixed=1000)
+def check_devnet_health(process):
+    if not process.is_alive:
+        return False
+
+    return nearlib.check_health()
+
+
+@pytest.fixture
+def make_devnet(request):
+    def _make_devnet(base_dir):
+        devnet_exe = os.environ['NEAR_DEVNET_EXE']
+        command = "{devnet_exe} -d {base_dir} --test-block-period 5" \
```

Comment:
Shouldn't we also:
* Use different rpc port for each test?
* Clean up the tmpdir?

Reply:
tmpdir is automatically cleaned up. i haven't had any issues with using the same port, because pytest kills the old devnet before the new one starts. i guess this cause spooky issues if termination wasn't happening properly.. will fix